Documents/UseCasesForWebsite.txt

```diff
@@ -1,9 +1,7 @@
 User 
 
-- user wants to see the info page 
-- user wants to apply as artist 
-- user wants to donate 
-- user wants to have a prostictic worked on
+- user wants to View info page, donate, request a prostetic or apply as a Artist 
```

Comment:
No 👎 . Please refer to David's go at it. This is very bad, because you actually have less info than in my sketch...

Give as much info as possible to every use case. And you are missing use cases as viewing gallery, liking a picture. Make it explicit that donations can be both from individuals and companies- that will be two different use cases (they start the same, but continue differently).

Also, there is no user. It's anonymous.

Reply:
oke, maybe I can better name it visitor then ? 
and I will look at David PR how he done it

Reply:
Visitor is a good name, yup.